tailwind.config.js

```diff
@@ -0,0 +1,35 @@
+import {readFileSync} from 'node:fs';
+import {parse} from 'css-variables-parser';
+
+const colors = Object.keys(parse([
+  readFileSync(new URL('web_src/css/themes/theme-gitea-light.css', import.meta.url), 'utf8'),
+  readFileSync(new URL('web_src/css/themes/theme-gitea-dark.css', import.meta.url), 'utf8'),
```

Comment:
I don't understand this section tbh.... Is it about importing our color namings? Then why isn't one theme enough?

Reply:
It reads our CSS variables to extract the color variables. The only reason I have both themes is in case someone would forget adding a color to one of them.

Reply:
:+1: